lib/src/layer/tile_layer.dart

```diff
@@ -429,6 +429,20 @@ class _TileLayerState extends State<TileLayer> with TickerProviderStateMixin {
       reloadTiles = true;
     }
 
+    var maxTileZoomValue = 1.0;
```

Comment:
if `reloadTiles == true` already, this section is unnecessary. Should be moved to `if(!reloadTiles) {...` on line 451